landing_page/src/components/contributors/contributors.js

```diff
@@ -32,13 +32,13 @@ function OrgLink(props) {
   let nameUrlRegex = /\s+\(.*\)\s?/i;
 
   if (org !== null) {
-    if (org.affiliation_id) {
+    if (org?.affiliation_id) {
```

Comment:
Are you using "optional chaining" here because "org is somethings "undefined"?  Since you already wrapping the code in "(org !== null)", I think you could add "&& org !== undefined" there, and then you would only need to use the optional chaining on the "affiliation_id".

Reply:
Thanks for addressing this Brian.